kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
+        kordac_extension = KordacExtension()
+        kordac_extension.heading = None
         self.required_files = {}
-        self.html_string = ''
-        html = None
-        ext = KordacExtension()
+
         converter = markdown.Markdown(extensions=[
```

Comment:
Is there any side effects you would be aware of by doing this? 

Didn't move it in case there was something you expected to behave differently. I added `kordac_extension.heading = None` to the `run` function to ensure it was set to `None` each time.